action.yaml

```diff
@@ -46,6 +46,10 @@ inputs:
   # create_hosted_zone:
   #   description: "Bool, whether the hosted zone should be created in the action. Default is false"
   #   default: false
+  create_vpc:
+    description: "Bool, whether a VPC should be created in the action. Default is false"
+    default: false
```

Comment:
This is helpful and makes a lot of sense :+1:
In prod users would probably manage VPC themselves, custom created or community module like https://registry.terraform.io/modules/terraform-aws-modules/vpc/aws/latest.

Looks like, however, the PR doesn't provide enough configuration to re-use the existing VPC or subnet specified by the user.

Reply:
yep! https://github.com/bitovi/github-actions-deploy-stackstorm/issues/4

want to add a readme and publish to GHA marketplace. @LeoDiazL is working on the `vpc_id` option